.github/workflows/unit.yml

```diff
@@ -18,8 +18,6 @@ jobs:
       uses: actions/setup-go@v2
       with:
         go-version: ${{ matrix.go }}
-    - name: Format
-      run: "find . -not \\( \\( -wholename './.git' \\) -prune \\) -name '*.go' | xargs gofmt -s -d"
```

Comment:
We can add windows to the test matrix now, yeah?

Reply:
Good point! Will try.

Reply:
Applied!